recipes-dom0/dom0-image-thin-initramfs/dom0-image-thin-initramfs.bbappend

```diff
@@ -15,6 +15,8 @@ python __anonymous () {
         d.appendVarFlag("do_compile", "depends", " domu-image-fusion:do_${BB_DEFAULT_TASK} ")
     if "domr" in guests :
         d.appendVarFlag("do_compile", "depends", " domu-image-litmusrt:do_${BB_DEFAULT_TASK} ")
+    if "domu" in guests :
```

Comment:
I think it is time to convert this succession of `if`s to something more data-driver:

```
guest_depends = {'domu':'domu-image-weston', 'domr':'domu-image-litmusrt`, ....}
for guest, dep in gust_depends.items():
   if guest in guests:
       d.appendVarFlag("do_compile", "depends", " " + dep  + ":do_${BB_DEFAULT_TASK} ")
```

Reply:
this is a refactoring which needs to be done later